docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -92,104 +36,194 @@ is an iteration of the last one, hence it improves the decision made by the
+members of a each class and the proportion that is misidentified.
+
+As the number of classes increases, the confusion matrix becomes more complex:
+
+image::images/confusion-matrix-multiclass.jpg[alt="Confusion matrix of a multiclass problem",width="100%",align="center"]
+
+This matrix contains the actual labels on the left side while the predicted 
+labels are on the top. The proportion of correct and incorrect predictions is 
+broken down for each class. This enables you to examine how the {classanalysis}
+confused the different classes while it made its predictions.
+
+
+[discrete]
+[[ml-dfanalytics-class-aucroc]]
+=== Area under the curve of receiver operating characteristic (AUC ROC)
+
+The receiver operating characteristic (ROC) curve is a plot that represents the 
+performance of the {classification} process at different predicted probability 
+thresholds. It compares the true positive rate for a specific class against the 
+rate of all the other classes combined ("one versus all" strategy) at the 
+different threshold levels to create the curve.
+
+.Example 
+You have three classes: `A`, `B`, and `C`, you want to calculate AUC ROC for 
+`A`. In this case, the number of correctly classified ++A++s (true positives) 
+are compared to the number of ++B++s and ++C++s that are misclassified as ++A++s 
+(false positives).
+
+From this plot, you can compute the area under the curve (AUC) value, which is a 
+number between 0 and 1. The higher the AUC, the better the model is at 
+predicting ++A++s as ++A++s, in this case.
 
-To improve the performance of your {classanalysis}, consider using a smaller 
-`training_percent` value when you create the job. That is to say, use a smaller 
-percentage of your documents to train the model more quickly. It is a good 
-strategy to make progress iteratively: use a smaller training percentage first, 
-run the analysis, and evaluate the performance. Then, based on the results, 
-decide if it is necessary to increase the `training_percent` value. If possible, 
-prepare your input data such that it has less classes. You can also remove the 
-fields that are not relevant from the analysis by specifying `excludes` patterns 
-in the `analyzed_fields` object when configuring the {dfanalytics-job}.  
+NOTE: To use this evaluation method, you must set `num_top_classes` to `-1`
+or a value greater than or equal to the total number of classes when you create
+the {dfanalytics-job}.
 
 
 [discrete]
-[[dfa-classification-interpret]]
-=== Interpreting {classification} results
+[[dfa-classification-feature-importance]]
+=== {feat-imp-cap}
 
-The following sections help you understand and interpret the results of a 
-{classanalysis}. To see example results, refer to
-<<flightdata-classification-results>>.
+{feat-imp-cap} provides further information about the results of an analysis and 
+helps to interpret the results in a more subtle way. If you want to learn more 
+about {feat-imp}, <<ml-feature-importance,click here>>. 
```

Comment:
Personally I prefer to use the page's title rather than having "click here":
```suggestion
about {feat-imp}, refer to <<ml-feature-importance>>. 
```